sklearn/kernel_approximation.py

```diff
@@ -110,6 +110,9 @@ def transform(self, X, y=None):
         projection *= np.sqrt(2.) / np.sqrt(self.n_components)
         return projection
 
+    def _get_tags(self):
+        return _update_tags(self, super(RBFSampler, self), stateless=True)
```

Comment:
Is this really stateless? It stores the random offset in fit and uses it in transform.

In general, how is this tag used / what is the plan for it? Things like a random state that initializes a rng in fit still counts as stateless?